src/theme/mui.ts

```diff
@@ -151,17 +152,9 @@ const theme = createTheme({
         fontFamily: secondaryFontFamily,
       },
     },
-    MuiFormHelperText: {
+    MuiFormControl: {
```

Comment:
Have you checked for `<FormControl>` components across the app? When you add new overrides, they will ofc affect all instances already present.

Reply:
This change replaces the `overflowStyle` that I deleted where the width was set inline to 100% and `overflow:hidden` which interferes with the new input labels. There was also a cryptic comment `// Neded for solving a fix in Windows browsers` so we should make sure to do a full regression test on all form elements. We can still use `width: 100%` instead of `flex: 1` as it is a bit more explicit.

Reply:
Make sure to add a comment about that to the testing section.